example_node:
    param1: 1
    param2: "test"
